torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1607,6 +1607,55 @@ def _make_tensor_helper(shape, low=None, high=None):
     return samples
 
 
+def scipy_reference_wrapper(ref):
```

Comment:
Clarify how and when to use this.

"If you're using a SciPy op as a reference, then pass it wrapped in a lambda to this function, which will either return the lambda or, if SciPy is unavailable, _NOTHING. Wrapping the operation in a lambda will prevent the Python interpreter from attempting to discover the operation, which will fail if SciPy is unavailable. See the OpInfo for digamma for an example."